frontend/src/lib/components/modals/ConfirmDeleteModal.svelte

```diff
@@ -17,6 +17,7 @@
   import { FormModal } from '$lib/components/modals';
   import type { FormModalResult, FormSubmitCallback } from '$lib/components/modals/FormModal.svelte';
   import { TrashIcon } from '$lib/icons';
+  import tt from '$lib/i18n';
```

Comment:
why `tt`?

Reply:
Because there's already a scoped-`t` and `globalT` is ugly. 🤷

Reply:
ugh, that is annoying. We should probably come up with a good solution for using scoped and global in the same file. But that's out of scope here.